.github/workflows/staging-build.yaml

```diff
@@ -5,9 +5,6 @@ on:
   push:
     tags:
       - '*'
-    branches:
```

Comment:
We do not want to build staging on every push to these branches, this is currently handled by the unstable builds. Once #4875 is complete this will be changed anyway.

Reply:
Apologies, meant to approve this rather than reject it. I thought I had changes in flight to handle this as well.

Reply:
Sorry, I missed that

Reply:
So, I reverted these lines. Is it OK or should I put them back?